src/command/Parser.java

```diff
@@ -185,17 +188,43 @@ private static void ProcessTodo(String input, TaskList tasklist, Ui ui){
      * @param tasklist Tasklist of the user.
      * @param ui Ui that interacts with the user.
      */
-    private static void ProcessEvent(String input, TaskList tasklist, Ui ui){
+    private static void ProcessDoAfter(String input, TaskList tasklist, Ui ui){
         try {
             String[] splitspace = input.split(" ", 2);
             String[] splitslash = splitspace[1].split("/", 2);
             String taskDescription = splitslash[0];
             String[] splittime = splitslash[1].split(" ", 2);
             String taskTime = splittime[1];
+            if (taskTime.contains("/")) {
             Date formattedtime = dataformat.parse(taskTime);
-            Event event = new Event(taskDescription, dataformat.format(formattedtime));
-            tasklist.AddTask(event);
-            ui.PrintAddedMessage(event, tasklist);
+            DoAfterTasks After = new DoAfterTasks(taskDescription, dataformat.format(formattedtime));
+            tasklist.AddTask(After);
+            ui.PrintAddedMessage(After, tasklist);
+            }
+            else{
+                DoAfterTasks After = new DoAfterTasks(taskDescription, taskTime);
+                tasklist.AddTask(After);
+                ui.PrintAddedMessage(After, tasklist);
+            }
+            }
```

Comment:
Perhaps moving this curly bracket one tab back to match the try block would help with readability.